src/panels/lovelace/editor/unused-entities/hui-unused-entities.ts

```diff
@@ -0,0 +1,209 @@
+import {
+  html,
+  LitElement,
+  TemplateResult,
+  PropertyValues,
+  property,
+  customElement,
+  css,
+  CSSResult,
+} from "lit-element";
+
+import { classMap } from "lit-html/directives/class-map";
+import "../../../../components/ha-fab";
+
+import "./hui-select-row";
+
+import { computeRTL } from "../../../../common/util/compute_rtl";
+import { computeUnusedEntities } from "../../common/compute-unused-entities";
+import { navigate } from "../../../../common/navigate";
+
+import { showSelectViewDialog } from "../select-view/show-select-view-dialog";
+import { showEditCardDialog } from "../card-editor/show-edit-card-dialog";
+
+import { HomeAssistant } from "../../../../types";
+import { Lovelace } from "../../types";
+import { LovelaceConfig } from "../../../../data/lovelace";
+
+@customElement("hui-unused-entities")
+export class HuiUnusedEntities extends LitElement {
+  @property() public lovelace?: Lovelace;
+
+  @property() public hass?: HomeAssistant;
+
+  @property() private _unusedEntities: string[] = [];
+
+  private _selectedEntities: string[] = [];
+
+  private get _config(): LovelaceConfig {
+    return this.lovelace!.config;
+  }
+
+  protected updated(changedProperties: PropertyValues): void {
+    super.updated(changedProperties);
+
+    if (changedProperties.has("lovelace")) {
+      if (
+        this.lovelace!.mode === "storage" &&
+        this.lovelace!.editMode === false
+      ) {
+        navigate(this, "/lovelace");
+        return;
+      }
+      this._getUnusedEntities();
+    }
+  }
+
+  protected render(): TemplateResult | void {
+    if (!this.hass || !this.lovelace) {
+      return html``;
+    }
+
+    if (this.lovelace.mode === "storage" && this.lovelace.editMode === false) {
+      return html``;
+    }
+
+    return html`
+      <ha-card header="Unused entities">
+        <div class="card-content">
+          These are the entities that you have available, but are not in your
+          Lovelace UI yet.
+          ${this.lovelace.mode === "storage"
+            ? html`
+                <br />Select the entities you want to add to a card and then
+                click the add card button.
+              `
+            : ""}
+        </div>
+        <table
+          class="entities"
+          @entity-selection-changed=${this._handleSelectionChanged}
+        >
+          <thead>
+            <tr>
+              <th>Entity</th>
+              <th>Entity id</th>
+              <th>Domain</th>
+              <th>Last changed</th>
+            </tr>
+          </thead>
+          <tbody>
+            ${this._unusedEntities.map((entity) => {
+              return html`
+                <hui-select-row
+                  .selectable=${this.lovelace!.mode === "storage"}
```

Comment:
I think that this is fine to get this PR merged, but we should add a new PR to add a dialog with selectable text to make this usable for YAML Mode. We could even make a generic dialog that we can offer YAML users in other places too.